WgSetup.py

```diff
@@ -8,85 +8,100 @@
 
 home_dir = os.path.expanduser('~')
 
-def wgdata(device, robot):
+# Format: 'Robot' : ["Public Key"],
+robot_table = {
+        'ram': 'PyCIhvAiBFNxP8Ka5MOhvqq9Q3LkBAddWBjlbb5HDUA=',
+        'tron': 'Public Key2',
+        'flynn': 'Public Key3'
+        }
+
+def findrobot(robot):
+    robot_lower = robot.lower()
+    if robot_lower not in robot_table:
+        print(f"{robot} not found. \n")
+        print("Available Robots: ")
+        for r in robot_table:
+            print(f"{r}".capitalize())
+        sys.exit(1)
+            
+            
+def wgdata(device):    
     home_dir = os.path.expanduser('~')
-    wg_data_dir = os.path.join(home_dir, 'wireguard_data')
+    wg_data_dir = os.path.join(home_dir, '.wireguard_data')
     
     # Check if .wireguard_data exists
     if not os.path.exists(wg_data_dir):
         os.mkdir(wg_data_dir)
         os.chmod(wg_data_dir, 0o700)
     
     # Create robot directory if it doesn't exist
-    robot_dir = os.path.join(wg_data_dir, robot)
+    robot_dir = os.path.join(wg_data_dir, robot.lower())
     if not os.path.exists(robot_dir):
         os.mkdir(robot_dir)
         os.chmod(robot_dir, 0o700)
     
     # Create device directory if it doesn't exist
-    device_dir = os.path.join(robot_dir, device)
+    device_dir = os.path.join(robot_dir, device.lower())
     if not os.path.exists(device_dir):
         os.mkdir(device_dir)
         os.chmod(device_dir, 0o700)
         
     return device_dir    
        
 
-def Generate(device):
+def Generate(robot, device):
+    # Check if robot exists
+    findrobot(robot)
```

Comment:
If this is the only call to findrobot, why not just inline the function here? It's pretty short, yes? Also, it's not normally good form to e.g. `exit(1)` from within a function call (though I've certainly done it).